src/www/ui/ui-export-list.php

```diff
@@ -522,6 +534,85 @@ private function printLines($lines, $copyright=false)
     return $V;
   }
 
+  /**
+   * Reduce license findings from agents into one
+   * @param array $lines     Scanned results of agents and conclusions
+   * @return array Lines with consolidated license list
+   */
+  private function consolidateResult($lines)
+  {
+    $newLines = [];
+    foreach ($lines as $row) {
+      $consolidatedLicenses = array();
+      if ($row['conclusions'] !== null) {
+        foreach ($row['agentFindings'] as $key => $value) {
+          unset($row['agentFindings'][$key]);
+        }
```

Comment:
This can be optimized as `$row['agentFindings'] = array();` if you want to delete everything inside `agentFindings`.

Reply:
Thanks @GMishx I just changed the code as you suggested.